README.md

```diff
@@ -40,6 +40,14 @@ To run the test-suite locally, first complete the bootstrap step as outline abov
 tox -e tests
 ```
 
+### Documentation
+
```

Comment:
Let's add a `#### Requirements` subsection here, mentioning you will need `python`, `tox`, and `plantuml` installed on your machine.